queries/useConversationListQuery.ts

```diff
@@ -207,7 +223,7 @@ const getConversationList = async (args: {
   }
 };
 
-export const conversationListQueryConfig = (args: {
+export const getConversationListQueryConfig = (args: {
```

Comment:
I think if we use the `select` pattern I mentioned earlier here, we could use a hook rather than a queryObserver for conversation list length - looking forward to chatting more shortly.